packages/@sanity/form-builder/src/inputs/Slug/SlugInput.tsx

```diff
@@ -31,6 +31,7 @@ type Props = {
   readOnly?: boolean
   document: SanityDocument
   onChange: (ev: PatchEvent) => void
+  onFocus: (pathOrEvent?: Path | React.SyntheticEvent | Event) => void
```

Comment:
This should be `onFocus: (path: Path) => void`

Reply:
OK, do you mean `onFocus([])` or `onFocus()`?

Reply:
Both should work, so `onFocus()` is probably better :)